api/pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache 2.0"
 
 [tool.poetry.dependencies]
 PyJWT = "2.4.0"
-python = "<3.11,>=3.8.0"
+python = ">=3.10,<3.11"
```

Comment:
rioxarray requires a python version greater than 3.10. This version update was generated by poetry based on all dependencys' requirements